File: Linguagem de Programação I/Extras/Beecrowd_Extra/2709.java
import java.util.Scanner;

public class Main {
    public static boolean is_prime(int n) {
        if (n < 2) {
            return false;
        }
        if (n == 2) {
            return true;
        }
        if (n % 2 == 0) {
            return false;
        }
        int p = 3;
        while (p * p <= n) {
            if (n % p == 0) {
                return false;
            }
            p += 2;
        }
        return true;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        while (scanner.hasNext()) {
            int M = scanner.nextInt();
            int[] coins = new int[M];
            for (int i = 0; i < M; i++) {
                coins[i] = scanner.nextInt();
            }
            int N = scanner.nextInt();
            int sum_coins = 0;
            for (int i = M - 1; i >= 0; i -= N) {
                sum_coins += coins[i];
            }
            if (is_prime(sum_coins)) {
                System.out.println("You’re a coastal aircraft, Robbie, a large silver aircraft.");
            } else {
                System.out.println("Bad boy! I’ll hit you.");
            }
        }
    }
}

File: Linguagem de Programação I/Extras/Beecrowd_Extra/2709.cs
using System;
using System.Collections.Generic;

class Program
{
    static bool IsPrime(int n)
    {
        if (n < 2) return false;
        if (n == 2) return true;
        if (n % 2 == 0) return false;
        for (int p = 3; p * p <= n; p += 2)
        {
            if (n % p == 0) return false;
        }
        return true;
    }

    static void Main(string[] args)
    {
        int M, N;
        while (int.TryParse(Console.ReadLine(), out M))
        {
            List<int> coins = new List<int>();
            for (int i = 0; i < M; i++)
            {
                coins.Add(int.Parse(Console.ReadLine()));
            }
            N = int.Parse(Console.ReadLine());
            int sumCoins = 0;
            for (int i = M - 1; i >= 0; i -= N)
            {
                sumCoins += coins[i];
            }
            if (IsPrime(sumCoins))
            {
                Console.WriteLine("You’re a coastal aircraft, Robbie, a large silver aircraft.");
            }
            else
            {
                Console.WriteLine("Bad boy! I’ll hit you.");
            }
        }
    }
}